src/layout/landingPage/Body.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
```

Comment:
Adding a top level comment on this file, but there is a lot of flexing going on here and we definitely don't need all of them. It just complicates things